lib/builderator/tasks/packer.rb

```diff
@@ -128,6 +128,42 @@ def wait(profile)
         say_status :complete, 'All copied images are available'
       end
 
+      desc 'remote_tag PROFILE', 'Apply existing tags to the AMI in remote AWS accounts'
+      def remote_tag(profile)
+        invoke :configure, [profile], options
+
+        sts_client = Aws::STS::Client.new(region: Config.aws.region)
+        allowed_cred_keys = %w(access_key_id secret_access_key session_token)
+
+        images.each do |image_name, (image, build)|
+          filters = [{
+            :name => 'name',
+            :values => [image_name]
+          }]
+
+          if build.tagging_role.nil?
+            puts "No remote tagging to be performed as no IAM role is defined"
+            return
+          end
+
+          build.ami_users.each do |account|
+            role_arn = "arn:aws:iam::#{account}:role/#{build.tagging_role}"
```

Comment:
I need to update the docs.  The idea would be you create the same role in each AWS account - Builderator will look for that one role everywhere...but this is completely up for discussion.